RELEASE/scripts/autoscend.ash

```diff
@@ -2409,6 +2409,9 @@ boolean LX_freeCombats(boolean powerlevel)
 		else
 		{
 			auto_log_debug("LX_freeCombats is calling neverendingPartyCombat()");
+			if (handleFamiliar($familiar[Red-Nosed Snapper])) {
```

Comment:
This line would hardcode always using the red-nosed snapper in the NEP if you have it.
is this done on purpose?

Reply:
Yes. Same as the other places this exact code is used (level 10 airship and various level 11 quest zones). Reason being human musk is pretty useful in-run so unless explicitly powerlevelling, it's probably worth charging 10/11 combats for 0 adventure cost.

Reply:
I see, thanks for clarifying.